scripts/install_renv_sysdeps.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -e
+
+## build ARGs
+NCPUS=${NCPUS:--1}
+
+# always set this for scripts but don't declare as ENV..
+export DEBIAN_FRONTEND=noninteractive
+export PATH=$PATH:/usr/local/texlive/bin/x86_64-linux/
+
+apt-get update
+
+#----- SysDeps for Tidyverse-----
+apt-get install -y --no-install-recommends \
+    file \
+    git \
+    libapparmor1 \
+    libgc1c2 \
+    libclang-dev \
+    libcurl4-openssl-dev \
+    libedit2 \
+    libobjc4 \
+    libssl-dev \
+    libpq5 \
+    lsb-release \
+    psmisc \
+    procps \
+    python-setuptools \
+    pwgen \
+    sudo \
+    wget
+
+#----- SysDeps for Verse-----
+apt-get install -y --no-install-recommends \
+    cmake \
+    default-jdk \
+    fonts-roboto \
+    ghostscript \
+    hugo \
+    less \
+    libbz2-dev \
+    libglpk-dev \
+    libgmp3-dev \
+    libfribidi-dev \
+    libharfbuzz-dev \
+    libhunspell-dev \
+    libicu-dev \
+    liblzma-dev \
+    libmagick++-dev \
+    libopenmpi-dev \
+    libpcre2-dev \
+    libv8-dev \
+    libxml2-dev\
+    libxslt1-dev \
+    libzmq3-dev \
+    qpdf \
+    texinfo \
+    software-properties-common \
+    vim
+
+# librdf0-dev depends on libcurl4-gnutils-dev instead of libcurl4-openssl-dev...
+# So: we can build the redland package bindings and then swap back to libcurl-openssl-dev... (ick)
+# explicitly install runtime library sub-deps of librdf0-dev so they are not auto-removed.
+apt-get install -y librdf0-dev
+install2.r --error --skipinstalled -n $NCPUS redland
```

Comment:
@cboettig Not sure what is needed in this section

Reply:
yeah, this is a bit of a mess.  debian/ubuntu ship a few different versions of libcurl (specifically, gnutils vs openssl), which conflict.  (Some packages can bind either one, but don't necessarily play nicely with both -- e.g. I believe `curl` R package doesn't fail but does some unexpected things with the non-openssl version iirc?)  

The `redland` R package needs `librdf0-dev` which depends on gnutils, meaning that line 64 will uninstalls the openssl version and potentially breaks things.  So if we're going to install `librdf0-dev`, I *think* it is necessary to reinstall `libcurl4-openssl-dev` after we finish building `redland` from source (maybe not necessary now that we can get binaries??)

Anyway, long story short, I've always been very confused about the libcurl conflict.  @eddelbuettel or @jeroen probably have a much better handle on this; maybe one of them could advise what's the best way to support both redland and curl R packages?

Reply:
Sorry, do not have a fix. There are alternative implementations, which in turn are use by different packages, so you could indeed end up with 'graph' with conflicts.  Best solution might be to locally rebuild packages using, say, gnutls to use openssl.  Not something I tried though.